go-ssb-bindings/ctrl.go

```diff
@@ -48,6 +48,21 @@ func ssbConnectPeer(quasiMs string) bool {
 		return false
 	}
 	level.Debug(log).Log("event", "dialed", "addr", msAddr.String())
+
+	if servicePlug == nil {
+		return true
+	}
+
+	go func() {
+		time.Sleep(10 * time.Second)
```

Comment:
Why the nap here?

Reply:
:+1: good spotting. This is not needed anymore.

The nap was needed to wait for `HandleCall` be processed and return the token.

Before I collected it to global var here, which is ugly of course and since we have the c function callback to notify this can go all together.